torchci/pages/api/drci/drci.ts

```diff
@@ -162,7 +163,30 @@ async function getBaseCommitJobs(
       jobsBySha.get(job.head_sha).set(job.name, job);
     }
   }
-  return jobsBySha;
+
+  const jobsByShaByName = new Map();
```

Comment:
is there a specific reason to not merge this part with the above loop that generates jobsBySha or is it to make it more readable?